packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -321,6 +321,26 @@ module.exports = {
         '/test_page/'
       ]
     },
+    'robots': {
+      host: 'https://developer.okta.com',
+      sitemap: "/docs-sitemap.xml",
```

Comment:
nit for consistency:
```suggestion
      sitemap: '/docs-sitemap.xml',
```

Reply:
done